src/olympia/abuse/models.py

```diff
@@ -873,10 +873,10 @@ class CinderPolicy(ModelBase):
     objects = CinderPolicyQuerySet.as_manager()
 
     def __str__(self):
-        return self.name
+        return self.full_text('')
 
     def full_text(self, canned_response_text=None):
-        if not canned_response_text:
+        if canned_response_text is None:
             canned_response_text = self.text
```

Comment:
This would mean that a policy from cinder would never be overridden by a ReviewerReason canned response, even if the canned response was blank.  (If we don't want that it'd need a tweak to ReasonsChoiceWidget)

Reply:
> I'm assuming all active `ReviewActionReason` do have a canned response anyway though.

This is true in prod atm.